### PR TITLE
update wrong reference to controls engineering in frc

### PR DESCRIPTION
**Pull request by laviRZ**

### CODE REVIEW

**Comment by jasondaming** [comment]
@calcmogul any other updates?

File: source/docs/software/advanced-controls/state-space/state-space-flywheel-walkthrough.rst [hunk 1]
```diff
@@ -95,7 +95,7 @@ The ``LinearSystem`` class contains methods for easily creating state-space syst
 Modeling Using Flywheel Moment of Inertia and Gearing
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A flywheel can also be modeled without access to a physical robot, using information about the motors, gearing and flywheel's :term:`moment of inertia`. A full derivation of this model is presented in Section 8.2.1 of  `Controls Engineering in FRC <https://file.tavsys.net/control/controls-engineering-in-frc.pdf>`__.
+A flywheel can also be modeled without access to a physical robot, using information about the motors, gearing and flywheel's :term:`moment of inertia`. A full derivation of this model is presented in Section 14.2 of  `Controls Engineering in FRC <https://file.tavsys.net/control/controls-engineering-in-frc.pdf>`__.
```
Comment (calcmogul) [inline source/docs/software/advanced-controls/state-space/state-space-flywheel-walkthrough.rst:98]: 14.2 is a link to 1-DOF feedforward, not the flywheel model derivation.

**Comment by calcmogul** [comment]
Should I make a new PR? The author seems to have abandoned it.

**Comment by laviRZ** [comment]
> Should I make a new PR? The author seems to have abandoned this one.

Sorry, I somehow missed your previous comment. Will implement your changes and update the PR👍

**Pull request closed without merging**